x/incentive/types/expected_keepers.go

```diff
@@ -94,9 +93,9 @@ type AccountKeeper interface {
 	GetModuleAccount(ctx sdk.Context, name string) authtypes.ModuleAccountI
 }
 
-// MintKeeper defines the required methods needed by this modules keeper
-type MintKeeper interface {
-	GetMinter(ctx sdk.Context) (minter minttypes.Minter)
+// KavamintKeeper defines the required methods needed by this modules keeper
+type KavamintKeeper interface {
+	GetStakingApy(ctx sdk.Context) sdk.Dec
```

Comment:
nit: is GetStakingApy a method on the kavaminter? It's generally more generic / powerful to return a true interface here versus trying to make golang bend over to create a bloated java class object with getters and setters....

Reply:
agreed. sadly though, getters like this appear to be the standard pattern for sharing values between modules (note that this interface is the kavamint keeper expected by x/incentive). i think the primary reason is because the actual value is dependent on context (block height to determine the correct parameter), not just some fixed value.

i only created getter-like methods on keepers for values that needed to be access by external modules like this one. the original x/mint GetMinter is a similar vein. as well as x/distribution GetCommunityTax used below.

it is a bit sad to write go like java :joy: :sob: but the pattern is strongly represented and is effective.

i suppose some benefits are more complex mocking powers for tests and the future allowance of `GetStakingApy` to be a more complex calculation. for now it's contents are indeed just `GetParams(ctx).StakingRewardsApy`...

Reply:
Naming aside, encapsulating the kavamint parameter structure is good here.  I'd rather add to the interface than couple external modules to the kava mint parameter structure.